tap_salesforce/__init__.py

```diff
@@ -417,5 +417,6 @@ def main():
         LOGGER.critical(e)
         sys.exit(1)
     except Exception as e:
-        LOGGER.critical(e)
```

Comment:
Will we get traceback after this change? In tap-facebook, they added Logger.exception specifically: https://github.com/singer-io/tap-facebook/blob/3318ad972a4c0df7b5f1c8bd57b942d5f6b39bf4/tap_facebook/__init__.py#L931

Reply:
Yes, we are getting traceback.

Reply:
As ask is to make it like a facebook, do we need to add Logger.exception specifically?

Reply:
I don't think so, the ask of this ticket was to Log `CRITICAL` for multiline error messages. And `LOGGER.exception` Logs `ERROR`.